src/ui/components/Collections/Collections.jsx

```diff
@@ -86,44 +91,73 @@ const Collections = props => {
       case '':
         return [];
       case 'servers':
-        headers = [{ value: 'icon', label: '' }, 'name', 'id', 'endpoint', 'type'];
+        headers = [
+          { value: 'icon', label: '', edit: false, viewOnly: true },
+          { value: 'name', label: 'name', edit: true },
+          { value: 'id', label: 'id', edit: false },
+          { value: 'endpoint', label: 'endpoint', edit: true },
+          { value: 'type', label: 'type', edit: false }
```

Comment:
Is there a reason this is false? When adding a new one there is no way to set this

Reply:
The designs show the type as uneditable, but I guess when adding a new one you do need to be able to add it.  I'll have to figure out how to resolve that.